lightcurve/views.py

```diff
@@ -9,6 +9,7 @@
 from imageflow.models import ImageAnalysis, Reduction, UserUploadedImage
 from lightcurve.models import LightCurve
 from reduction.util import find_point_by_id
+import csv
```

Comment:
Let's move this import up top (followed by new line), because it is from the standard python library.

Reply:
Done